src/domUpdates.js

```diff
@@ -0,0 +1,89 @@
+const myTrips = document.querySelector('#myTrips');
+const pastTrips = document.querySelector('#pastTrips');
+const upcomingTrips = document.querySelector('#upcomingTrips');
+const pendingTrips = document.querySelector('#pendingTrips');
+const amountSpentLastYear = document.querySelector('#amountSpentLastYear');
+
+
+const updateDom = (user, destinations) => {
+  populateDashboard(user, destinations);
+}
+
+const populateDashboard = (user, destinations) => {
+  displayTripArticles(user, destinations);
+  displayAmountSpentThisYear(user);
+}
+
+const displayTripArticles = (user, destinations) => {
+  let today = new Date();
+  today = [today.getFullYear(), today.getMonth() + 1, today.getDate()].reduce((acc, val) => {
+    val += '';
+    if (val.length === 1) {
+      val = '0' + val;
+    }
+    return [...acc, val];
+  }, []).join('/');
+
+  let travelerPastTrips = user.trips.filter(trip => trip.date <= today);
```

Comment:
Very interesting on that reverse fat arrow, I would love to know the background on that.